src/tracing_conf.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This project sits separately right? Add Rainier repo as a dependency ? Or you are just suggesting to follow the structure of bainbridge?  Also, if both of them are running on the same server wouldn't it become a default **localhost**?